src/DependencyManagement/DependencyManager.cs

```diff
@@ -269,16 +252,66 @@ internal void InstallFunctionAppDependencies(PowerShell pwsh, ILogger logger)
             finally
             {
                 // Clean up
-                pwsh.AddCommand(Utils.RemoveModuleCmdletInfo)
-                    .AddParameter("Name", "PackageManagement, PowerShellGet")
-                    .AddParameter("Force", Utils.BoxedTrue)
-                    .AddParameter("ErrorAction", "SilentlyContinue")
-                    .InvokeAndClearCommands();
+                RemoveSaveModuleModules(pwsh);
             }
         }
 
         #region Helper_Methods
 
+        /// <summary>
+        /// Runs Save-Module which downloads a module locally from the specified repository.
+        /// </summary>
+        protected virtual void RunSaveModuleCommand(PowerShell pwsh, string repository, string moduleName, string version, string path)
+        {
+            pwsh.AddCommand(SaveModuleCmdletName)
```

Comment:
You should follow RemoveModuleCmdletInfo as an exam ple and get the cmdletinfo for Save-Module as well. It's a small perf gain

Reply:
@TylerLeonhardt : Thanks for the suggestion. 
I just had a quick sync with @daxian-dbw , and we cannot create a cmdlet info object for Save-Help since it is a script cmdlet.

Reply:
`Save-Module` is a function, and `FunctionInfo` doesn't have a public constructor.

Reply:
Ah yes. Good point. Nevermind then.